### ISSUE TITLE
Weak password protection

### ISSUE THREAD

**Issue by VMbuseck** [bhdouglass/owncloud-files-scope#6]
From a short view to the code:
1) Password clearly visible
2) Password stored in plain text
3) Password transmitted in cleartext to the server within the URL

**Comment by bhdouglass** [comment]
I'm currently looking to using the built in "Online Accounts" to better protect the password. I do not believe that the password is transmitted in cleartext, that's just a standard way of setting it (there is some magic that should transform it into a base64 http header). So as long as you are using https the password it self should not be visible over the network.

**Comment by VMbuseck** [comment]
Your answer to 3) is correct. The transmission starts with the 'PROPFIND' method and the payload is completely encrypted, if you are using the https protocol.

**Comment by bhdouglass** [comment]
Yes HTTPS is very recommended.

**Comment by ernesst** [comment]
Is there any plan or bug raise to have Owncloud account within UT ?

**Comment by bhdouglass** [comment]
Yes I have it on my to do list and this is kinda the bug for it.

**Comment by VMbuseck** [comment]
4) Error message contains password in plain text

If an error occurs, the error message contains the password in plain text:

Error message: request to https://user:pass@server.com/remote.php/webdav// failed, reason: self signed certificate.

**Comment by bhdouglass** [comment]
Sorry about that, I'll get that fixed soon.

**Comment by bhdouglass** [comment]
I've just fixed the error messages containing the username/password in version 1.7 and it also moves the username/password out of the url and into the header.
